### PR TITLE
#11: Implementing what happens when clicked on poi

### PR DESCRIPTION
**Pull request by viveanban**
*WIP*
- Created a onClick listener to handle what happens when the POI is clicked
- Created an annotation (TypeDef) to define the POI types.
- Renamed the png files to reflect the POI types

_If you want to test this, you can replace the LocationFragment in the main activity with the POIFragment_

### CODE REVIEW

File: app/src/main/java/com/example/concordia_campus_guide/Models/PoiType.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.example.concordia_campus_guide.Models;
+
+//TODO: Make sure these strings correspond exactly to the PointTypes
```
Comment (viveanban) [inline app/src/main/java/com/example/concordia_campus_guide/Models/PoiType.java]: Before completing this, we will need to change the PointType into a typeDef ([for better performance](https://android.jlelse.eu/android-performance-avoid-using-enum-on-android-326be0794dc3)). The todo is created as a reminder (an improvement issue will probably be created too).

Comment (ferdousara-parvin) [reply app/src/main/java/com/example/concordia_campus_guide/Models/PoiType.java]: Yeah create an improvement issue asap in order to not forget

**Comment by ferdousara-parvin** [comment]
Tested it, works great! :)

File: app/src/main/java/com/example/concordia_campus_guide/Models/PoiType.java (hunk 1 above)
Comment (viveanban) [reply app/src/main/java/com/example/concordia_campus_guide/Models/PoiType.java]: Done! [#147]

**Pull request merged**